skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -529,46 +509,45 @@ public function executeEditBox() {
 
 		// Editing: foo
 		$this->displaytitle = true;
-		$this->title = wfMsg( $msg, htmlspecialchars( $wgTitle->getPrefixedText() ) );
+		$this->title = wfMessage( $msg, $this->wg->Title->getPrefixedText() )->parse();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)